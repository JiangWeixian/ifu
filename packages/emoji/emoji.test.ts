import alfyTest from 'alfy-test'
import { test, expect } from 'vitest'

test('should has default output', async () => {
  const alfy = alfyTest()

  const result = await alfy('love')
  expect(result).toMatchSnapshot()
})
