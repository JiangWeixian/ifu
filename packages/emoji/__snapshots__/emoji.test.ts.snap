// Vitest Snapshot v1

exports[`should has default output 1`] = `
[
  {
    "arg": "🥰",
    "icon": {
      "path": " ",
    },
    "subtitle": "smiling face with hearts",
    "text": {
      "copy": "🥰",
      "largetype": "🥰",
    },
    "title": "🥰 :smiling_face_with_three_hearts:",
  },
  {
    "arg": "😍",
    "icon": {
      "path": " ",
    },
    "subtitle": "smiling face with heart-eyes",
    "text": {
      "copy": "😍",
      "largetype": "😍",
    },
    "title": "😍 :heart_eyes:",
  },
  {
    "arg": "💘",
    "icon": {
      "path": " ",
    },
    "subtitle": "heart with arrow",
    "text": {
      "copy": "💘",
      "largetype": "💘",
    },
    "title": "💘 :cupid:",
  },
  {
    "arg": "❤️",
    "icon": {
      "path": " ",
    },
    "subtitle": "red heart",
    "text": {
      "copy": "❤️",
      "largetype": "❤️",
    },
    "title": "❤️ :heart:",
  },
  {
    "arg": "💌",
    "icon": {
      "path": " ",
    },
    "subtitle": "love letter",
    "text": {
      "copy": "💌",
      "largetype": "💌",
    },
    "title": "💌 :love_letter:",
  },
  {
    "arg": "🤟",
    "icon": {
      "path": " ",
    },
    "subtitle": "love-you gesture",
    "text": {
      "copy": "🤟",
      "largetype": "🤟",
    },
    "title": "🤟 :love_you_gesture:",
  },
  {
    "arg": "🍀",
    "icon": {
      "path": " ",
    },
    "subtitle": "four leaf clover",
    "text": {
      "copy": "🍀",
      "largetype": "🍀",
    },
    "title": "🍀 :four_leaf_clover:",
  },
  {
    "arg": "🏩",
    "icon": {
      "path": " ",
    },
    "subtitle": "love hotel",
    "text": {
      "copy": "🏩",
      "largetype": "🏩",
    },
    "title": "🏩 :love_hotel:",
  },
  {
    "arg": "🥊",
    "icon": {
      "path": " ",
    },
    "subtitle": "boxing glove",
    "text": {
      "copy": "🥊",
      "largetype": "🥊",
    },
    "title": "🥊 :boxing_glove:",
  },
  {
    "arg": "🧤",
    "icon": {
      "path": " ",
    },
    "subtitle": "gloves",
    "text": {
      "copy": "🧤",
      "largetype": "🧤",
    },
    "title": "🧤 :gloves:",
  },
]
`;
